src/Comps/search/Search.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { SearchIcon } from "../../Icons";
+import {
+  SearchFormContainer,
+  SearchButton,
+  Input,
+  VerticalLine,
+} from "./style";
+// import DropDown from '../dropdown/DropDown';
+
+const Search = () => {
+  return (
```

Comment:
Think about what else you need in the component like a state that holds the input value and an onChange function.
prepare anything you can before you actually use the real data.
Also read about Debounce search and try implementing it.